frontend/src/App.tsx

```diff
@@ -3,6 +3,8 @@ import theme from './theme';
 import {Route, Routes} from 'react-router-dom';
 import Home from './pages/home';
 import GoogleAuthLogin from './components/GoogleAuthLogin';
+import Footer from './components/Footer/footer.tsx';
```

Comment:
Instead of importing like this, you can do something like
`import Footer from './components/Footer';`
Figure out how you can achieve this.